.travis.yml

```diff
@@ -14,5 +14,6 @@ install: true
 before_install:
   - ./tools/travis/setup.sh
 
+  - openssl aes-256-cbc -K $encrypted_fddf8425fc8b_key -iv $encrypted_fddf8425fc8b_iv -in credentials.json.enc -out tests/credentials.json -d
```

Comment:
@lzbj 
Where are the keys like encrypted_fddf8425fc8b_key and encrypted_fddf8425fc8b_iv?
I have downloaded your patch and submit it to my own repo, it still raises the same errors.
I think the problem is due to the missing secret keys, you have called along this line.